mergify_engine/actions/merge.py

```diff
@@ -546,9 +547,8 @@ def merge_report(
         return check_api.Result(conclusion, title, summary)
 
     @staticmethod
-    def get_queue_summary(ctxt):
-        q = queue.Queue.from_context(ctxt)
-        pulls = q.get_pulls()
+    def get_queue_summary(ctxt: context.Context, queue: queue.Queue) -> str:
```

Comment:
```suggestion
    def get_queue_summary(ctxt: context.Context, q: queue.Queue) -> str:
```
or whatever that does not shadow the `queue` module

Reply:
done